NOTICE.txt

```diff
@@ -8,14 +8,13 @@ Licensed under the Apache License, Version 2.0 (the "License"); you may not use
  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 
 This project includes:
+  aircompressor under Apache License 2.0
```

Comment:
@bhasudha are these legit changes? if not, can you please backout changes to NOTICE.txt across this PR?

Reply:
Done!